Tests/BoundariesTests/__Snapshots__/BoundariesTests.testCombinedBoundaries.txt

```diff
@@ -1,64 +1,78 @@
+          ▿ parallel: 2 elements
+            ▿ Boundaries.Cmd<BoundariesTests.MyEffect>.execute
+              ▿ execute: BoundariesTests.MyEffect.print
+                - print: "incr"
+            ▿ Boundaries.Cmd<BoundariesTests.MyEffect>.execute
+              ▿ execute: BoundariesTests.MyEffect.trackAnalytics
+                ▿ trackAnalytics: (2 elements)
+                  - eventName: "Incremented"
+                  ▿ properties: 1 key/value pair
+                    ▿ (2 elements)
+                      - key: "count"
+                      - value: 1
   ▿ (4 elements)
     - message: ""
-    ▿ action: Optional(BoundariesTests.AppAction.counterAction(BoundariesTests.CounterAction.decr))
-      ▿ some: BoundariesTests.AppAction.counterAction
-        - counterAction: BoundariesTests.CounterAction.decr
+    ▿ action: BoundariesTests.AppAction.counterAction
+      - counterAction: BoundariesTests.CounterAction.decr
     ▿ state: BoundariesTests.AppState
       ▿ counter: BoundariesTests.CounterState
         - count: 0
       - loggedIn: true
-    ▿ effect: Boundaries.Effect<BoundariesTests.AppAction>.sequence
+    ▿ effect: Boundaries.Cmd<BoundariesTests.MyEffect>.sequence
       ▿ sequence: 2 elements
-        ▿ Boundaries.Effect<BoundariesTests.AppAction>.batch
-          - batch: 0 elements
-        ▿ Boundaries.Effect<BoundariesTests.AppAction>._execute
-          ▿ _execute: (2 elements)
-            ▿ .0: 2 elements
-              - "print"
-              - "decr"
-            - .1: (Function)
+        ▿ Boundaries.Cmd<BoundariesTests.MyEffect>.parallel
+          - parallel: 0 elements
+        ▿ Boundaries.Cmd<BoundariesTests.MyEffect>.execute
+          ▿ execute: BoundariesTests.MyEffect.print
+            - print: "decr"
   ▿ (4 elements)
     - message: ""
-    ▿ action: Optional(BoundariesTests.AppAction.settingsAction)
-      - some: BoundariesTests.AppAction.settingsAction
+    - action: BoundariesTests.AppAction.settingsAction
     ▿ state: BoundariesTests.AppState
       ▿ counter: BoundariesTests.CounterState
         - count: 0
       - loggedIn: false
-    ▿ effect: Boundaries.Effect<BoundariesTests.AppAction>.sequence
+    ▿ effect: Boundaries.Cmd<BoundariesTests.MyEffect>.sequence
       ▿ sequence: 2 elements
-        ▿ Boundaries.Effect<BoundariesTests.AppAction>._execute
-          ▿ _execute: (2 elements)
-            - .0: "Clear Cache"
-            - .1: (Function)
-        ▿ Boundaries.Effect<BoundariesTests.AppAction>.batch
-          - batch: 0 elements
+        ▿ Boundaries.Cmd<BoundariesTests.MyEffect>.execute
+          ▿ execute: BoundariesTests.MyEffect.request
+            - request: "http://www.google.com"
+        ▿ Boundaries.Cmd<BoundariesTests.MyEffect>.parallel
+          - parallel: 0 elements
+  ▿ (4 elements)
+    - message: ""
+    ▿ action: BoundariesTests.AppAction.requestResult
+      - requestResult: "http://www.google.com: HELLO WORLD!"
```

Comment:
this action is a cool example of a testable effect. so the previous action `.settingsAction` emitted an `.execute(.request("http://www.google.com"))` effect, and our test store ran it immediately with a mocked executor, which then dispatched this `.requestResult("http://www.google.com: HELLO WORLD!")` right back into the store.